shopify/marketShopify.js

```diff
@@ -2,282 +2,333 @@ isMobile = function() {
+    var temp = "https://cdn.shopify.com/s/files/1/0384/0265/files/" + mageName + "?1619266020467081722$$REVISION$$";
+    document.getElementById(imageName).src = temp;
+    var mageText = "#" + imageName + "Cont p";
+    document.querySelector(mageText).innerHTML = mage;
+  };
+
+  var imageName = "boss0";
+  var fBosses = bosses.filter(boundInExp);
+  var chosenBoss = _.sample(fBosses) + ".jpg";
+  var temp = "https://cdn.shopify.com/s/files/1/0384/0265/files/" + chosenBoss + "?14241145229160548124$$REVISION$$"
+  document.getElementById(imageName).src= temp;
+};
+
+createInput = function(type, value, name, checked = false) {
+  var input = document.createElement("input");
+  input.type = type;
+  input.name = name;
+  input.value = value;
+  input.checked = checked;
+
+  return input;
 };
 
-function buttonPress() {
-	//document.getElementById('image4').scrollIntoView();
-	var checkExp = document.forms[0];
-	exps = ["Base"];
-	for (i = 0; i < checkExp.length; i++) {
-        if (checkExp[i].checked) {
-            exps.push(checkExp[i].value);
-        };
-    };
-	var fGems = cardsGem.filter(inExp);
-	var fRelics = cardsRelic.filter(inExp);
-	var fSpells = cardsSpell.filter(inExp);
-	var numMages = 4;
-	var fMages = mages.filter(inExp);
-    var checkSetup = document.forms[1];
-    var txt = "";
-	var cards = [];
-    for (i = 0; i < checkSetup.length; i++) {
-        if (checkSetup[i].checked) {
-            txt = txt + checkSetup[i].value + " ";
-			cards = genMarket(fGems, fRelics, fSpells, i);
-        };
-    };
-    var chosenMages = _.sample(fMages,4)
-    for (i = 0; i < cards.length; i++) {
-		var imageName = "image"+ i.toString();
-        document.getElementById(imageName).src= cards[i] +".jpg";
-        //document.getElementById(imageName).width="300"; // example of how you can alter width of an image
-    };	
-    for (i = 0; i < numMages; i++) {
-		var imageName = "mage"+ i.toString();
-        document.getElementById(imageName).src= chosenMages[i] +".jpg";
-        //document.getElementById(imageName).width="300"; // example of how you can alter width of an image
-    };
-    var imageName = "boss0";
-    var fBosses = bosses.filter(inExp);
-    var chosenBoss = _.sample(fBosses);
-    document.getElementById(imageName).src= chosenBoss +".jpg";
-    //document.getElementById("market").innerHTML = "You ordered a market with: " + txt;
-	//document.getElementById("expo").innerHTML = "You want to use expansions: " + exps+fGems;
-	
+createImage = function(imageName, width) {
+  var img = document.createElement("img");
+  img.src = "images/" + imageName + ".jpg";
+  img.style.width = width;
+
+  return img;
+};
+
+createExpBoxes = function() {
+  var ae1 = new Map([["AE", "AeonsEnd"], ["Depths", "TheDepths"], ["Nameless", "TheNameless"]]);
+  var ae2 = new Map([["WE", "WarEternal"], ["TV", "TheVoid"], ["OD", "TheOuterDark"]]);
+  var ae3 = new Map([["Legacy", "Legacy"], ["BS", "BuriedSecrets"]]);
+  var waves = [ae1, ae2]; //, ae3];
```

Comment:
Re-add `ae3` to this list to add Legacy. This should also make it easier for you to add AE4 content later